app/controllers/stories_controller.rb

```diff
@@ -358,22 +360,16 @@ def set_user_json_ld
         "@id": URL.user(@user)
       },
       "url": URL.user(@user),
-      "sameAs": [],
+      "sameAs": user_same_as,
       "image": ProfileImage.new(@user).get(width: 320),
       "name": @user.name,
-      "email": "",
-      "jobTitle": "",
+      "email": @user.email.presence,
```

Comment:
this can only be shown if the email is public, see https://github.com/thepracticaldev/dev.to/pull/7731/files#diff-036c389e944275ed9596b5d2c9b30027L444

Reply:
Good catch! I can add a check for the public email here.